packages/builder/src/schematics/ng-add/index.ts

```diff
@@ -68,7 +88,7 @@ export function builder(options: any): Rule {
 
 export default function (options: any): Rule {
   return chain([
-    mergeWith(apply(url("./template"), [move(normalize("./src"))])),
+    mergeWith(apply(url("./template"), [move(normalize("./src"))]), MergeStrategy.Overwrite),
```

Comment:
I think it is better not to overwrite the user files.

Reply:
if a user has already env.d.ts file in his src, the schematics fails without the "override" strategy...
Using git you can always revert undesired changes

Reply:
let me know what to do

Reply:
ah I see but what if someone has updated angular.json for example without committing it would his modifications be overridden/lost?

Reply:
yes..., maybe in some way we could control if the repository is up-to-date without pending changes before running schematics (as angular do, but idk how)